bedrock/products/urls.py

```diff
@@ -19,17 +20,6 @@
     page("vpn/mobile/", "products/vpn/platforms/mobile.html", ftl_files=["products/vpn/platforms/mobile", "products/vpn/shared"]),
     page("vpn/mobile/ios/", "products/vpn/platforms/ios.html", ftl_files=["products/vpn/platforms/ios", "products/vpn/shared"]),
     page("vpn/mobile/android/", "products/vpn/platforms/android.html", ftl_files=["products/vpn/platforms/android", "products/vpn/shared"]),
-    # Evergreen SEO articles (issue #10224)
-    page("vpn/more/what-is-an-ip-address/", "products/vpn/more/ip-address.html", ftl_files=["products/vpn/more/ip-address", "products/vpn/shared"]),
-    page("vpn/more/vpn-or-proxy/", "products/vpn/more/vpn-or-proxy.html", ftl_files=["products/vpn/more/vpn-or-proxy", "products/vpn/shared"]),
-    page("vpn/more/what-is-a-vpn/", "products/vpn/more/what-is-a-vpn.html", ftl_files=["products/vpn/more/what-is-a-vpn", "products/vpn/shared"]),
-    page(
-        "vpn/more/when-to-use-a-vpn/", "products/vpn/more/when-to-use.html", ftl_files=["products/vpn/more/when-to-use-a-vpn", "products/vpn/shared"]
```

Comment:
Nit: Can these FTL files and their configs be removed?

- https://github.com/mozilla/bedrock/tree/main/l10n/en/products/vpn/more
- https://github.com/mozilla/bedrock/tree/main/l10n/en/products/vpn/platforms
- https://github.com/mozilla/bedrock/blob/main/l10n/configs/pontoon.toml#L324-L329